<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorDialog</class>
 <widget class="QMainWindow" name="EditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">QtJsonEditor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QJsonWidget" name="widget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>369</width>
           <height>488</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_schema">
            <property name="text">
             <string>schema</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_updateSchema">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Update widget</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit_schema">
          <property name="lineWrapMode">
           <enum>QTextEdit::NoWrap</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_json">
            <property name="text">
             <string>json</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_updateJsonValue">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Update widget</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit_json">
          <property name="lineWrapMode">
           <enum>QTextEdit::NoWrap</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>814</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLangue">
    <property name="title">
     <string>&amp;Language</string>
    </property>
    <addaction name="actionEnglish"/>
    <addaction name="actionFrench"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpenSchema"/>
    <addaction name="actionOpenJson"/>
    <addaction name="actionSaveSchema"/>
    <addaction name="actionSaveJson"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuLangue"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionEnglish">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionFrench">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>French</string>
   </property>
  </action>
  <action name="actionOpenSchema">
   <property name="text">
    <string>Open Schema file</string>
   </property>
  </action>
  <action name="actionOpenJson">
   <property name="text">
    <string>&amp;Open Json file</string>
   </property>
  </action>
  <action name="actionSaveJson">
   <property name="text">
    <string>&amp;Save Json file</string>
   </property>
  </action>
  <action name="actionSaveSchema">
   <property name="text">
    <string>Save Schema file</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QJsonWidget</class>
   <extends>QWidget</extends>
   <header>QtJsonWidget/QJsonWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
